modules/ROOT/pages/actions/legacy-endpoints.adoc

```diff
@@ -0,0 +1,65 @@
+:description: Legacy endpoints.
+
+[[http-api-legacy-endpoints]]
+= Legacy Endpoints label:deprecated[]
+
+[IMPORTANT]
+====
+The API described in this section of the manual has been deprecated and will be removed in Neo4j 5.0.
+====
+
+For backwards compatibility with neo4j 3.5, it is still possible to
+use the HTTP API without specifying the database in the URL. For example:
+
+`http://localhost:7474/db/data/transaction/commit`.
+
+Since the database is not specified, the statements will be executed against the user's _home database_.
```

Comment:
just a note on cherry-picking back to pre-homedb that the database the cypher is executed against is the configured global default database